docs/docsource/installation/upgrade/xdm_migrate_from_kubedb_to_bitnami.rst

```diff
@@ -0,0 +1,282 @@
+service from KubeDB to Bitnami by following these manual procedures.
+
+.. note::
+
+   In the following documentation, `artesca` is the name of the |product|
+   instance, and the migration is intended for |product| version 2.0.1 and
+   earlier to 2.x.x.
+
+   The commands are written with the |product|
+   instance running Metalk8s, and the namespace is `zenko` and the
+   instance name is `artesca`.
+
+Before Upgrade
+==============
+
+#. Make sure all resources are in a ``Running`` state.
+
+#. Download the ISOs:
+
+   .. code:: 
+
+      curl -u <user> -L -o /srv/scality/release/zenko-base-2.x.x.iso \
+         https://...
+      curl -u <user> -L -o /srv/scality/release/zenko-2.x.x.iso \
+         https://...
+
+#. Import and activate ``zenko-base`` solution:
+
+   .. code::
+
+      /srv/scality/metalk8s-2.9.2/solution.sh import -a /srv/scality/release/zenko-base-2.x.x.iso
+      /srv/scality/metalk8s-2.9.2/solution.sh activate -n zenko-base -V 2.x.x
+
+#. Import and activate ``zenko`` solution:
+
+   .. code::
+
+      /srv/scality/metalk8s-2.9.2/solution.sh import -a /srv/scality/release/zenko-2.x.x.iso
+      /srv/scality/metalk8s-2.9.2/solution.sh activate -n zenko -V 2.x.x
+
+Upgrade
+=======
+
+The upgrade procedure will consist of:
+
+- Deleting the KubeDB MongoDB deployment and freeing the PV resources used.
+- Deploying MongoDB using the Bitnami charts.
+- Updating the existing Zenko instance to use the Bitnami MongoDB.
+- Updating the `MongoDB` replica set configuration.
+
+.. warning::
+
+   The upgrade procedure will result in |product| instance downtime, and the
+   following services will not be operational:
+
+   - Cloudserver S3 APIs
+   - Backbeat APIs
+   - Orbit Management APIs
+   - Vault Management APIs
+
+Update the ``Zenko-Operator``
+-----------------------------
+
+#. Update the ``zenko-operator`` deployment:
+
+   .. code::
+
+      /srv/scality/metalk8s-2.9.2/solution.sh add-solution -n zenko -s zenko -V 2.x.x
+
+   After a few seconds you should see the new operator running. From there you can update each components.
+
+Deploy the ``Bitnami MongoDB``
+------------------------------
+
+#. Create the `secret` resource manifest, ``mongodb-creds.yaml``
+
+   .. code:: 
+
+      MONGODB_ROOT_USERNAME=$(kubectl -n zenko get secret artesca-data-base-db-auth -o jsonpath='{.data.username}')
+      MONGODB_ROOT_PASSWORD=$(kubectl -n zenko get secret artesca-data-base-db-auth -o jsonpath='{.data.password}')
+      MONGODB_USERNAME=$(kubectl -n zenko get secret artesca-data-db-app-creds.v2 -o jsonpath='{.data.username}')
+      MONGODB_PASSWORD=$(kubectl -n zenko get secret artesca-data-db-app-creds.v2 -o jsonpath='{.data.password}')
+      MONGODB_REPLICA_SET_KEY=$(kubectl -n zenko get secret artesca-data-base-db-cert -o jsonpath='{.data.key\.txt}')
+      INSTANCE_ID=$(kubectl --kubeconfig /etc/kubernetes/admin.conf -n zenko get zenko/artesca-data -o jsonpath='{.status.instanceID}' | base64)
```

Comment:
If the `--kubeconfig` option is needed here, it would be good to use it in the previous commands as well.

Reply:
* removed `--kubeconfig`  to be consistent with all the other commands